storage/tianmu/core/engine.cpp

```diff
@@ -1093,8 +1093,12 @@ static void HandleDelayedLoad(int tid, std::vector<std::unique_ptr<char[]>> &vec
     dbname.length=db_name.length();
     loadquery.str=const_cast<char *>(load_data_query.c_str());
     loadquery.length=load_data_query.length();
-    tabname.str=const_cast<char *>(tab_name.c_str());
-    tabname.length=tab_name.length();
+    
+    //Failed to insert data into a table with a table name with special characters(#362)
```

Comment:
`//Failed to insert data into a table with a table name with special characters(#362)`
--->
`// fix issue 362: bug for insert into a table which table name contains special characters`